minigunapp/views.py

```diff
@@ -0,0 +1,62 @@
+import json
+import yaml
+
+from django.http import HttpResponse, Http404, JsonResponse
+from django.shortcuts import render
+from cerberus import Validator
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import authentication, permissions
+
+from minigunapp.models import Email
+from minigunapp.serializers import EmailSerializer
+from minigunapp.tasks import send_email
+
+
+v = Validator()
+schema = yaml.load(open('minigunapp/schemas/email.yaml'))
+page_size = 10
+
+
+class EmailList(APIView):
+
+    def get(self, request, format=None):
+        """ Returns a single page of emails in Json format """
+
+        page = int(request.GET.get('page', 1))
+        page = page if page > 0 else 1
+        start = page_size * page
+        end = page_size * (page + 1)
+
+        emails = Email.objects[start:end]
+        emails = [EmailSerializer(email).data for email in emails]
```

Comment:
I apologize if the question wasn't stated clearly. It was more about how to handle actual change from the customer perspective. They might already have their code built around the API and it will be impossible just to add a field in the response in this particular case because an array is being returned. One of the solutions could be to add a a new version of the API.